2018-04-19T18:36:42.9317431+01:00 0HLD63GEOELUP:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:5000/   (ca22a1cb)
2018-04-19T18:36:43.4467431+01:00 0HLD63GEOELUP:00000001 [INF] Executing action method "Tinix.Controllers.HomeController.Index (Tinix)" with arguments ([""]) - ModelState is Valid (ba7f4ac2)
2018-04-19T18:37:09.4867431+01:00 0HLD63GEOELUP:00000001 [INF] Executing ViewResult, running view at path "/Views/Home/Index.cshtml". (9707eebe)
2018-04-19T18:37:10.9897431+01:00 0HLD63GEOELUP:00000001 [INF] Executed action "Tinix.Controllers.HomeController.Index (Tinix)" in 27588.6084ms (afa2e885)
2018-04-19T18:37:11.0447431+01:00 0HLD63GEOELUQ:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:5000/css/site.css   (ca22a1cb)
2018-04-19T18:37:11.0577431+01:00 0HLD63GEOELUR:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:5000/js/popper.min.js   (ca22a1cb)
2018-04-19T18:37:11.0677431+01:00 0HLD63GEOELUP:00000001 [INF] Request finished in 28082.729ms 200 text/html; charset=utf-8 (791a596a)
2018-04-19T18:37:11.0747431+01:00 0HLD63GEOELUS:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:5000/css/bootstrap.min.css   (ca22a1cb)
2018-04-19T18:37:11.1087431+01:00 0HLD63GEOELUR:00000001 [INF] Sending file. Request path: '"/js/popper.min.js"'. Physical path: '"C:\Users\ondero\desktop\dev\tinix\src\Tinix\wwwroot\js\popper.min.js"' (27b0a520)
2018-04-19T18:37:11.1137431+01:00 0HLD63GEOELUQ:00000001 [INF] Sending file. Request path: '"/css/site.css"'. Physical path: '"C:\Users\ondero\desktop\dev\tinix\src\Tinix\wwwroot\css\site.css"' (27b0a520)
2018-04-19T18:37:11.1397431+01:00 0HLD63GEOELUS:00000001 [INF] Sending file. Request path: '"/css/bootstrap.min.css"'. Physical path: '"C:\Users\ondero\desktop\dev\tinix\src\Tinix\wwwroot\css\bootstrap.min.css"' (27b0a520)
2018-04-19T18:37:11.1687431+01:00 0HLD63GEOELUT:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:5000/js/jquery.js   (ca22a1cb)
2018-04-19T18:37:11.1767431+01:00 0HLD63GEOELUT:00000001 [INF] Sending file. Request path: '"/js/jquery.js"'. Physical path: '"C:\Users\ondero\desktop\dev\tinix\src\Tinix\wwwroot\js\jquery.js"' (27b0a520)
2018-04-19T18:37:11.2147431+01:00 0HLD63GEOELUU:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:5000/js/bootstrap.min.js   (ca22a1cb)
2018-04-19T18:37:11.2257431+01:00 0HLD63GEOELUQ:00000001 [INF] Request finished in 206.9651ms 200 text/css (791a596a)
2018-04-19T18:37:11.2777431+01:00 0HLD63GEOELUR:00000001 [INF] Request finished in 168.8698ms 200 application/javascript (791a596a)
2018-04-19T18:37:11.2887431+01:00 0HLD63GEOELUU:00000001 [INF] Sending file. Request path: '"/js/bootstrap.min.js"'. Physical path: '"C:\Users\ondero\desktop\dev\tinix\src\Tinix\wwwroot\js\bootstrap.min.js"' (27b0a520)
2018-04-19T18:37:11.3127431+01:00 0HLD63GEOELUU:00000001 [INF] Request finished in 113.7484ms 200 application/javascript (791a596a)
2018-04-19T18:37:11.3517431+01:00 0HLD63GEOELUS:00000001 [INF] Request finished in 256.2612ms 200 text/css (791a596a)
2018-04-19T18:37:11.3797431+01:00 0HLD63GEOELUT:00000001 [INF] Request finished in 251.6987ms 200 application/javascript (791a596a)
2018-04-19T18:37:14.8917431+01:00 0HLD63GEOELUT:00000002 [INF] Request starting HTTP/1.1 GET http://localhost:5000/favicon.ico   (ca22a1cb)
2018-04-19T18:37:14.8937431+01:00 0HLD63GEOELUT:00000002 [INF] Sending file. Request path: '"/favicon.ico"'. Physical path: '"C:\Users\ondero\desktop\dev\tinix\src\Tinix\wwwroot\favicon.ico"' (27b0a520)
2018-04-19T18:37:14.8967431+01:00 0HLD63GEOELUT:00000002 [INF] Request finished in 6.2914ms 200 image/x-icon (791a596a)
2018-04-19T19:05:23.0567240+01:00 0HLD640EAE339:00000001 [ERR] An unhandled exception has occurred: One or more compilation failures occurred:
C:\Users\ondero\Desktop\dev\Tinix\src\Tinix\Views\Shared\_PostDetail.cshtml(20,30): error CS1061: 'BlogPostComment' does not contain a definition for 'comment' and no extension method 'comment' accepting a first argument of type 'BlogPostComment' could be found (are you missing a using directive or an assembly reference?) (8df55373)
Microsoft.AspNetCore.Mvc.Razor.Compilation.CompilationFailedException: One or more compilation failures occurred:
C:\Users\ondero\Desktop\dev\Tinix\src\Tinix\Views\Shared\_PostDetail.cshtml(20,30): error CS1061: 'BlogPostComment' does not contain a definition for 'comment' and no extension method 'comment' accepting a first argument of type 'BlogPostComment' could be found (are you missing a using directive or an assembly reference?)
   at Microsoft.AspNetCore.Mvc.Razor.Internal.RazorViewCompiler.CompileAndEmit(RazorCodeDocument codeDocument, String generatedCode)
   at Microsoft.AspNetCore.Mvc.Razor.Internal.RazorViewCompiler.CompileAndEmit(String relativePath)
   at Microsoft.AspNetCore.Mvc.Razor.Internal.RazorViewCompiler.CreateCacheEntry(String normalizedPath)
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at Microsoft.AspNetCore.Mvc.Razor.Internal.DefaultRazorPageFactoryProvider.CreateFactory(String relativePath)
   at Microsoft.AspNetCore.Mvc.Razor.RazorViewEngine.CreateCacheResult(HashSet`1 expirationTokens, String relativePath, Boolean isMainPage)
   at Microsoft.AspNetCore.Mvc.Razor.RazorViewEngine.OnCacheMiss(ViewLocationExpanderContext expanderContext, ViewLocationCacheKey cacheKey)
   at Microsoft.AspNetCore.Mvc.Razor.RazorViewEngine.LocatePageFromViewLocations(ActionContext actionContext, String pageName, Boolean isMainPage)
   at Microsoft.AspNetCore.Mvc.Razor.RazorViewEngine.FindView(ActionContext context, String viewName, Boolean isMainPage)
   at Microsoft.AspNetCore.Mvc.ViewEngines.CompositeViewEngine.FindView(ActionContext context, String viewName, Boolean isMainPage)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.HtmlHelper.<RenderPartialCoreAsync>d__60.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.HtmlHelper.<PartialAsync>d__57.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at Microsoft.AspNetCore.Mvc.Rendering.HtmlHelperPartialExtensions.Partial(IHtmlHelper htmlHelper, String partialViewName, Object model, ViewDataDictionary viewData)
   at Microsoft.AspNetCore.Mvc.Rendering.HtmlHelperPartialExtensions.Partial(IHtmlHelper htmlHelper, String partialViewName, Object model)
   at AspNetCore._Views_Home_Detail_cshtml.<ExecuteAsync>d__0.MoveNext() in C:\Users\ondero\Desktop\dev\Tinix\src\Tinix\Views\Home\Detail.cshtml:line 5
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.<RenderPageCoreAsync>d__16.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.<RenderPageAsync>d__15.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.<RenderAsync>d__14.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.<ExecuteAsync>d__22.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.<ExecuteAsync>d__21.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.ViewResult.<ExecuteResultAsync>d__26.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeResultAsync>d__19.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeNextResultFilterAsync>d__24.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResultExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeNextResourceFilter>d__22.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeFilterPipelineAsync>d__17.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeAsync>d__15.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Builder.RouterMiddleware.<Invoke>d__4.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>d__6.MoveNext()
2018-04-19T22:16:58.2899226+01:00 0HLD67BGTP6RI:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:5000/   (ca22a1cb)
2018-04-19T22:16:59.0779677+01:00 0HLD67BGTP6RI:00000001 [INF] Executing action method "Tinix.Controllers.HomeController.Index (Tinix)" with arguments ([""]) - ModelState is Valid (ba7f4ac2)
2018-04-19T22:17:03.8372399+01:00 0HLD67BGTP6RI:00000001 [INF] Executing ViewResult, running view at path "/Views/Home/Index.cshtml". (9707eebe)
2018-04-19T22:17:05.7813511+01:00 0HLD67BGTP6RI:00000001 [INF] Executed action "Tinix.Controllers.HomeController.Index (Tinix)" in 6734.7423ms (afa2e885)
2018-04-19T22:17:05.8373543+01:00 0HLD67BGTP6RI:00000001 [INF] Request finished in 7527.183ms 200 text/html; charset=utf-8 (791a596a)
2018-04-19T22:17:05.8473549+01:00 0HLD67BGTP6RJ:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:5000/css/site.css   (ca22a1cb)
2018-04-19T22:17:05.8493550+01:00 0HLD67BGTP6RI:00000002 [INF] Request starting HTTP/1.1 GET http://localhost:5000/css/bootstrap.min.css   (ca22a1cb)
2018-04-19T22:17:05.8573554+01:00 0HLD67BGTP6RK:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:5000/js/jquery.js   (ca22a1cb)
2018-04-19T22:17:05.8733563+01:00 0HLD67BGTP6RJ:00000001 [INF] The file "/css/site.css" was not modified (f1f8d725)
2018-04-19T22:17:05.8943575+01:00 0HLD67BGTP6RI:00000002 [INF] The file "/css/bootstrap.min.css" was not modified (f1f8d725)
2018-04-19T22:17:05.9303596+01:00 0HLD67BGTP6RK:00000001 [INF] The file "/js/jquery.js" was not modified (f1f8d725)
2018-04-19T22:17:05.9363599+01:00 0HLD67BGTP6RJ:00000001 [INF] Request finished in 49.4567ms 304 text/css (791a596a)
2018-04-19T22:17:05.9503607+01:00 0HLD67BGTP6RI:00000002 [INF] Request finished in 49.8252ms 304 text/css (791a596a)
2018-04-19T22:17:05.9533609+01:00 0HLD67BGTP6RI:00000003 [INF] Request starting HTTP/1.1 GET http://localhost:5000/js/popper.min.js   (ca22a1cb)
2018-04-19T22:17:05.9573611+01:00 0HLD67BGTP6RJ:00000002 [INF] Request starting HTTP/1.1 GET http://localhost:5000/js/bootstrap.min.js   (ca22a1cb)
2018-04-19T22:17:05.9693618+01:00 0HLD67BGTP6RI:00000003 [INF] The file "/js/popper.min.js" was not modified (f1f8d725)
2018-04-19T22:17:05.9703619+01:00 0HLD67BGTP6RJ:00000002 [INF] The file "/js/bootstrap.min.js" was not modified (f1f8d725)
2018-04-19T22:17:05.9843627+01:00 0HLD67BGTP6RI:00000003 [INF] Request finished in 19.3002ms 304 application/javascript (791a596a)
2018-04-19T22:17:05.9873629+01:00 0HLD67BGTP6RK:00000001 [INF] Request finished in 85.6888ms 304 application/javascript (791a596a)
2018-04-19T22:17:05.9893630+01:00 0HLD67BGTP6RJ:00000002 [INF] Request finished in 33.3729ms 304 application/javascript (791a596a)
2018-04-19T22:17:15.5589103+01:00 0HLD67BGTP6RJ:00000003 [INF] Request starting HTTP/1.1 GET http://localhost:5000/admin/login   (ca22a1cb)
2018-04-19T22:17:15.5739112+01:00 0HLD67BGTP6RJ:00000003 [INF] Executing action method "Tinix.Controllers.AdminController.Login (Tinix)" with arguments (null) - ModelState is Valid (ba7f4ac2)
2018-04-19T22:17:16.2679509+01:00 0HLD67BGTP6RJ:00000003 [INF] Executing ViewResult, running view at path "/Views/Admin/Login.cshtml". (9707eebe)
2018-04-19T22:17:16.7469783+01:00 0HLD67BGTP6RJ:00000003 [INF] Executed action "Tinix.Controllers.AdminController.Login (Tinix)" in 1170.9186ms (afa2e885)
2018-04-19T22:17:16.7519786+01:00 0HLD67BGTP6RJ:00000003 [INF] Request finished in 1191.316ms 200 text/html; charset=utf-8 (791a596a)
2018-04-19T22:17:16.8149822+01:00 0HLD67BGTP6RJ:00000004 [INF] Request starting HTTP/1.1 GET http://localhost:5000/css/admin.css   (ca22a1cb)
2018-04-19T22:17:16.8219826+01:00 0HLD67BGTP6RJ:00000004 [INF] The file "/css/admin.css" was not modified (f1f8d725)
2018-04-19T22:17:16.8259828+01:00 0HLD67BGTP6RJ:00000004 [INF] Request finished in 11.6184ms 304 text/css (791a596a)
2018-04-19T22:17:16.8399836+01:00 0HLD67BGTP6RI:00000004 [INF] Request starting HTTP/1.1 GET http://localhost:5000/js/jquery.validate.min.js   (ca22a1cb)
2018-04-19T22:17:16.8429838+01:00 0HLD67BGTP6RI:00000004 [INF] The file "/js/jquery.validate.min.js" was not modified (f1f8d725)
2018-04-19T22:17:16.8449839+01:00 0HLD67BGTP6RI:00000004 [INF] Request finished in 9.0863ms 304 application/javascript (791a596a)
2018-04-19T22:17:16.8779858+01:00 0HLD67BGTP6RI:00000005 [INF] Request starting HTTP/1.1 GET http://localhost:5000/js/jquery.validate.unobtrusive.js   (ca22a1cb)
2018-04-19T22:17:16.8839861+01:00 0HLD67BGTP6RI:00000005 [INF] The file "/js/jquery.validate.unobtrusive.js" was not modified (f1f8d725)
2018-04-19T22:17:16.8999870+01:00 0HLD67BGTP6RI:00000005 [INF] Request finished in 9.1344ms 304 application/javascript (791a596a)
2018-04-19T22:17:19.6441440+01:00 0HLD67BGTP6RI:00000006 [INF] Request starting HTTP/1.1 POST http://localhost:5000/Admin/Login application/x-www-form-urlencoded 33 (ca22a1cb)
2018-04-19T22:17:19.7371493+01:00 0HLD67BGTP6RI:00000006 [INF] Executing action method "Tinix.Controllers.AdminController.Login (Tinix)" with arguments (["Tinix.Models.LoginViewModel"]) - ModelState is Valid (ba7f4ac2)
2018-04-19T22:25:32.9883617+01:00 0HLD67G7844D7:00000003 [ERR] An unhandled exception has occurred: Object reference not set to an instance of an object. (e9c253af)
System.NullReferenceException: Object reference not set to an instance of an object.
   at AspNetCore._Views_Admin_Login_cshtml.<<ExecuteAsync>b__9_0>d.MoveNext() in C:\Users\ozan\Desktop\projects\Tinix\src\Tinix\Views\Admin\Login.cshtml:line 31
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Razor.Runtime.TagHelpers.TagHelperExecutionContext.<GetChildContentAsync>d__31.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at Microsoft.AspNetCore.Mvc.TagHelpers.RenderAtEndOfFormTagHelper.<ProcessAsync>d__7.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Razor.Runtime.TagHelpers.TagHelperRunner.<RunAsync>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at AspNetCore._Views_Admin_Login_cshtml.<ExecuteAsync>d__9.MoveNext() in C:\Users\ozan\Desktop\projects\Tinix\src\Tinix\Views\Admin\Login.cshtml:line 12
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.<RenderPageCoreAsync>d__16.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.<RenderPageAsync>d__15.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.<RenderAsync>d__14.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.<ExecuteAsync>d__22.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.<ExecuteAsync>d__21.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.ViewResult.<ExecuteResultAsync>d__26.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeResultAsync>d__19.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeNextResultFilterAsync>d__24.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResultExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeNextResourceFilter>d__22.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeFilterPipelineAsync>d__17.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeAsync>d__15.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Builder.RouterMiddleware.<Invoke>d__4.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>d__6.MoveNext()
2018-04-19T22:25:52.9795051+01:00 0HLD67G7844DA:00000004 [ERR] An unhandled exception has occurred: Object reference not set to an instance of an object. (e9c253af)
System.NullReferenceException: Object reference not set to an instance of an object.
   at AspNetCore._Views_Admin_Login_cshtml.<<ExecuteAsync>b__9_0>d.MoveNext() in C:\Users\ozan\Desktop\projects\Tinix\src\Tinix\Views\Admin\Login.cshtml:line 31
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Razor.Runtime.TagHelpers.TagHelperExecutionContext.<GetChildContentAsync>d__31.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at Microsoft.AspNetCore.Mvc.TagHelpers.RenderAtEndOfFormTagHelper.<ProcessAsync>d__7.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Razor.Runtime.TagHelpers.TagHelperRunner.<RunAsync>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at AspNetCore._Views_Admin_Login_cshtml.<ExecuteAsync>d__9.MoveNext() in C:\Users\ozan\Desktop\projects\Tinix\src\Tinix\Views\Admin\Login.cshtml:line 12
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.<RenderPageCoreAsync>d__16.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.<RenderPageAsync>d__15.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.<RenderAsync>d__14.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.<ExecuteAsync>d__22.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.<ExecuteAsync>d__21.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.ViewResult.<ExecuteResultAsync>d__26.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeResultAsync>d__19.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeNextResultFilterAsync>d__24.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResultExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeNextResourceFilter>d__22.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeFilterPipelineAsync>d__17.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeAsync>d__15.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Builder.RouterMiddleware.<Invoke>d__4.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>d__6.MoveNext()
2018-04-19T22:29:27.8237935+01:00 0HLD67IGD878G:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:5000/   (ca22a1cb)
2018-04-19T22:29:28.2188161+01:00 0HLD67IGD878G:00000001 [INF] Executing action method "Tinix.Controllers.HomeController.Index (Tinix)" with arguments ([""]) - ModelState is Valid (ba7f4ac2)
2018-04-19T22:29:32.1080385+01:00 0HLD67IGD878G:00000001 [INF] Executing ViewResult, running view at path "/Views/Home/Index.cshtml". (9707eebe)
2018-04-19T22:29:33.9291427+01:00 0HLD67IGD878G:00000001 [INF] Executed action "Tinix.Controllers.HomeController.Index (Tinix)" in 5746.9132ms (afa2e885)
2018-04-19T22:29:33.9791455+01:00 0HLD67IGD878G:00000001 [INF] Request finished in 6142.7573ms 200 text/html; charset=utf-8 (791a596a)
2018-04-19T22:29:34.2641618+01:00 0HLD67IGD878G:00000002 [INF] Request starting HTTP/1.1 GET http://localhost:5000/favicon.ico   (ca22a1cb)
2018-04-19T22:29:34.2761625+01:00 0HLD67IGD878G:00000002 [INF] Sending file. Request path: '"/favicon.ico"'. Physical path: '"C:\Users\ozan\desktop\projects\tinix\src\Tinix\wwwroot\favicon.ico"' (27b0a520)
2018-04-19T22:29:34.3001639+01:00 0HLD67IGD878G:00000002 [INF] Request finished in 39.7326ms 200 image/x-icon (791a596a)
2018-04-19T22:29:39.5144621+01:00 0HLD67IGD878G:00000003 [INF] Request starting HTTP/1.1 GET http://localhost:5000/admin/login   (ca22a1cb)
2018-04-19T22:29:39.5394636+01:00 0HLD67IGD878G:00000003 [INF] Executing action method "Tinix.Controllers.AdminController.Login (Tinix)" with arguments (null) - ModelState is Valid (ba7f4ac2)
2018-04-19T22:29:40.1454982+01:00 0HLD67IGD878G:00000003 [INF] Executing ViewResult, running view at path "/Views/Admin/Login.cshtml". (9707eebe)
2018-04-19T22:29:40.2085018+01:00 0HLD67IGD878G:00000003 [INF] Executed action "Tinix.Controllers.AdminController.Login (Tinix)" in 675.353ms (afa2e885)
2018-04-19T22:29:40.3505100+01:00 0HLD67IGD878G:00000003 [ERR] An unhandled exception has occurred while executing the request (cf926119)
System.NullReferenceException: Object reference not set to an instance of an object.
   at AspNetCore._Views_Admin_Login_cshtml.<<ExecuteAsync>b__9_0>d.MoveNext() in C:\Users\ozan\desktop\projects\tinix\src\Tinix\Views\Admin\Login.cshtml:line 31
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Razor.Runtime.TagHelpers.TagHelperExecutionContext.<GetChildContentAsync>d__31.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at Microsoft.AspNetCore.Mvc.TagHelpers.RenderAtEndOfFormTagHelper.<ProcessAsync>d__7.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Razor.Runtime.TagHelpers.TagHelperRunner.<RunAsync>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at AspNetCore._Views_Admin_Login_cshtml.<ExecuteAsync>d__9.MoveNext() in C:\Users\ozan\desktop\projects\tinix\src\Tinix\Views\Admin\Login.cshtml:line 12
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.<RenderPageCoreAsync>d__16.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.<RenderPageAsync>d__15.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.<RenderAsync>d__14.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.<ExecuteAsync>d__22.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.<ExecuteAsync>d__21.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.ViewResult.<ExecuteResultAsync>d__26.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeResultAsync>d__19.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeNextResultFilterAsync>d__24.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResultExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeNextResourceFilter>d__22.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeFilterPipelineAsync>d__17.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeAsync>d__15.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Builder.RouterMiddleware.<Invoke>d__4.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.<Invoke>d__7.MoveNext()
2018-04-19T22:29:40.5755228+01:00 0HLD67IGD878G:00000003 [INF] Request finished in 1065.1309ms 500 text/html; charset=utf-8 (791a596a)
2018-04-19T22:32:18.1065331+01:00 0HLD67K37JT86:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:5000/   (ca22a1cb)
2018-04-19T22:32:18.4535529+01:00 0HLD67K37JT86:00000001 [INF] Executing action method "Tinix.Controllers.HomeController.Index (Tinix)" with arguments ([""]) - ModelState is Valid (ba7f4ac2)
2018-04-19T22:32:22.9188083+01:00 0HLD67K37JT86:00000001 [INF] Executing ViewResult, running view at path "/Views/Home/Index.cshtml". (9707eebe)
2018-04-19T22:32:24.7479130+01:00 0HLD67K37JT86:00000001 [INF] Executed action "Tinix.Controllers.HomeController.Index (Tinix)" in 6335.901ms (afa2e885)
2018-04-19T22:32:24.7639139+01:00 0HLD67K37JT87:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:5000/css/bootstrap.min.css   (ca22a1cb)
2018-04-19T22:32:24.8079164+01:00 0HLD67K37JT88:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:5000/css/site.css   (ca22a1cb)
2018-04-19T22:32:24.8229172+01:00 0HLD67K37JT89:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:5000/js/jquery.js   (ca22a1cb)
2018-04-19T22:32:24.8489187+01:00 0HLD67K37JT88:00000001 [INF] The file "/css/site.css" was not modified (f1f8d725)
2018-04-19T22:32:24.8539190+01:00 0HLD67K37JT89:00000001 [INF] The file "/js/jquery.js" was not modified (f1f8d725)
2018-04-19T22:32:24.8589193+01:00 0HLD67K37JT87:00000001 [INF] The file "/css/bootstrap.min.css" was not modified (f1f8d725)
2018-04-19T22:32:24.8949214+01:00 0HLD67K37JT86:00000001 [INF] Request finished in 6680.5572ms 200 text/html; charset=utf-8 (791a596a)
2018-04-19T22:32:24.9079221+01:00 0HLD67K37JT89:00000001 [INF] Request finished in 84.9797ms 304 application/javascript (791a596a)
2018-04-19T22:32:24.9099222+01:00 0HLD67K37JT87:00000001 [INF] Request finished in 139.3192ms 304 text/css (791a596a)
2018-04-19T22:32:24.9299234+01:00 0HLD67K37JT88:00000001 [INF] Request finished in 127.0756ms 304 text/css (791a596a)
2018-04-19T22:32:24.9549248+01:00 0HLD67K37JT87:00000002 [INF] Request starting HTTP/1.1 GET http://localhost:5000/js/bootstrap.min.js   (ca22a1cb)
2018-04-19T22:32:24.9759260+01:00 0HLD67K37JT88:00000002 [INF] Request starting HTTP/1.1 GET http://localhost:5000/js/popper.min.js   (ca22a1cb)
2018-04-19T22:32:24.9819263+01:00 0HLD67K37JT87:00000002 [INF] The file "/js/bootstrap.min.js" was not modified (f1f8d725)
2018-04-19T22:32:24.9859266+01:00 0HLD67K37JT88:00000002 [INF] The file "/js/popper.min.js" was not modified (f1f8d725)
2018-04-19T22:32:24.9909269+01:00 0HLD67K37JT88:00000002 [INF] Request finished in 53.7409ms 304 application/javascript (791a596a)
2018-04-19T22:32:24.9939270+01:00 0HLD67K37JT87:00000002 [INF] Request finished in 72.3139ms 304 application/javascript (791a596a)
2018-04-19T22:32:33.6664231+01:00 0HLD67K37JT88:00000003 [INF] Request starting HTTP/1.1 GET http://localhost:5000/admin/login   (ca22a1cb)
2018-04-19T22:32:33.6854242+01:00 0HLD67K37JT88:00000003 [INF] Executing action method "Tinix.Controllers.AdminController.Login (Tinix)" with arguments (null) - ModelState is Valid (ba7f4ac2)
2018-04-19T22:32:34.2224549+01:00 0HLD67K37JT88:00000003 [INF] Executing ViewResult, running view at path "/Views/Admin/Login.cshtml". (9707eebe)
2018-04-19T22:32:34.6644802+01:00 0HLD67K37JT88:00000003 [INF] Executed action "Tinix.Controllers.AdminController.Login (Tinix)" in 981.8753ms (afa2e885)
2018-04-19T22:32:34.7084827+01:00 0HLD67K37JT88:00000003 [INF] Request finished in 1009.5808ms 200 text/html; charset=utf-8 (791a596a)
2018-04-19T22:32:34.8514908+01:00 0HLD67K37JT87:00000003 [INF] Request starting HTTP/1.1 GET http://localhost:5000/js/jquery.validate.min.js   (ca22a1cb)
2018-04-19T22:32:34.8754922+01:00 0HLD67K37JT89:00000002 [INF] Request starting HTTP/1.1 GET http://localhost:5000/js/jquery.validate.unobtrusive.js   (ca22a1cb)
2018-04-19T22:32:34.8764923+01:00 0HLD67K37JT88:00000004 [INF] Request starting HTTP/1.1 GET http://localhost:5000/css/admin.css   (ca22a1cb)
2018-04-19T22:32:34.8824926+01:00 0HLD67K37JT87:00000003 [INF] The file "/js/jquery.validate.min.js" was not modified (f1f8d725)
2018-04-19T22:32:34.8884930+01:00 0HLD67K37JT88:00000004 [INF] The file "/css/admin.css" was not modified (f1f8d725)
2018-04-19T22:32:34.8954934+01:00 0HLD67K37JT88:00000004 [INF] Request finished in 51.5167ms 304 text/css (791a596a)
2018-04-19T22:32:34.8954934+01:00 0HLD67K37JT87:00000003 [INF] Request finished in 49.2551ms 304 application/javascript (791a596a)
2018-04-19T22:32:34.8984935+01:00 0HLD67K37JT89:00000002 [INF] The file "/js/jquery.validate.unobtrusive.js" was not modified (f1f8d725)
2018-04-19T22:32:34.9024938+01:00 0HLD67K37JT89:00000002 [INF] Request finished in 59.1765ms 304 application/javascript (791a596a)
2018-04-19T22:32:39.0177291+01:00 0HLD67K37JT89:00000003 [INF] Request starting HTTP/1.1 POST http://localhost:5000/Admin/Login application/x-www-form-urlencoded 33 (ca22a1cb)
2018-04-19T22:32:39.1147347+01:00 0HLD67K37JT89:00000003 [INF] Executing action method "Tinix.Controllers.AdminController.Login (Tinix)" with arguments (["Tinix.Models.LoginViewModel"]) - ModelState is Valid (ba7f4ac2)
